pmlc/rt/timed_executable.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Intel Corporation
+
+#include "pmlc/rt/timed_executable.h"
+
+#include <chrono>
+#include <utility>
+
+#include "pmlc/util/logging.h"
+
+namespace pmlc::rt {
+namespace {
+
+// A trivial stopwatch class.
+//
+// TODO: Consider using std::chrono::high_resolution_clock.  Also consider
```

Comment:
Steady clock is better for this sort of thing.

Reply:
IIRC, on Windows, they're the same.  :-)

Reply:
Can we drop this TODO then? I don't think there's anything else to consider.

Reply:
Done